lib/src/navbar_router.dart

```diff
@@ -216,6 +242,25 @@ class _NavbarRouterState extends State<NavbarRouter>
     NavbarNotifier.makeBadgeVisible(NavbarNotifier.currentIndex, true);
     initAnimation();
     NavbarNotifier.index = widget.initialIndex;
+
+    // necessary init for swipeable
+    _pageController = PageController(initialPage: widget.initialIndex);
+
+    NavbarNotifier.addIndexChangeListener(
```

Comment:
You should also dispose this listener by calling `NavbarNotifier.removeLastListener` in dispose